com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -998,6 +1003,10 @@ static void InitializeAdditionalCameraData(Camera camera, UniversalAdditionalCam
             renderingData.perObjectData = GetPerObjectLightFlags(renderingData.lightData.additionalLightsCount);
             renderingData.postProcessingEnabled = anyPostProcessingEnabled;
 
+            //pipelineCommandBuffer = CommandBufferPool.Get();
+            //renderingData.commandBuffer = pipelineCommandBuffer;
+            renderingData.commandBuffer = null; // initialize after this?
```

Comment:
This got me questioning if we should get a command buffer per camera or one per frame. It seems like if you initialize in rendering data, then it should be per-frame?